tap_google_sheets/transform.py

```diff
@@ -104,6 +105,10 @@ def transform_sheet_boolean_data(value, sheet_title, col_name, col_letter, col_t
             col_val = True
         elif value.lower() in ('false', 'f', 'no', 'n'):
             col_val = False
+        elif value in ('1', '-1', '1.00', '-1.00'):
```

Comment:
Do add comments to explain this condition.

Reply:
Added comments